### PR TITLE
DC-1612: Bump SDK version to 2.1.15

### PR DESCRIPTION
**Pull request by hung-nguyen-hoang**

### CODE REVIEW

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/545/ : SUCCESS in 4s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-lcm-build-pipeline/542/ : FAILURE in 4m 09s

**Comment by hung-nguyen-hoang** [comment]
ok to test

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/546/ : SUCCESS in 7s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-lcm-build-pipeline/543/ : SUCCESS in 20m 19s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/548/ : SUCCESS in 4s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-lcm-build-pipeline/545/ : SUCCESS in 21m 09s

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,7 @@
 # GoodData Ruby SDK Changelog
+## 2.1.15
+ - FEATURE: BCO-1119 Introduce data sources API
+
```
Comment (danh-ung) [inline CHANGELOG.md:4]: We should mention Rails 6.1 support:
Upgrade activesupport to be able used with Rails 6.1

Comment (hung-nguyen-hoang) [reply CHANGELOG.md:5]: Done update change logs

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- gooddata-ruby-sonarqube-scan-zuul-el7 : SKIPPED (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-merge-compliance-check/489/ : SUCCESS in 14s
- https://checklist.intgdc.com/job/lcm/job/gooddata-ruby-travis-check/138/ : FAILURE in 7s
- gooddata-ruby-fossa-scanning-zuul-el7 : SKIPPED (non-voting)

**Comment by hung-nguyen-hoang** [comment]
ok to test

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/550/ : SUCCESS in 57s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-lcm-build-pipeline/547/ : FAILURE in 4m 45s

**Comment by hung-nguyen-hoang** [comment]
ok to test

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-oss-check/551/ : SUCCESS in 5s
- https://jenkins-ii.intgdc.com/job/gooddata-ruby/job/gooddata-ruby-lcm-build-pipeline/548/ : SUCCESS in 21m 07s

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- gooddata-ruby-sonarqube-scan-zuul-el7 : SKIPPED (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-merge-compliance-check/490/ : SUCCESS in 24s
- https://checklist.intgdc.com/job/lcm/job/gooddata-ruby-travis-check/139/ : FAILURE in 19s
- gooddata-ruby-fossa-scanning-zuul-el7 : SKIPPED (non-voting)

**Comment by yenkins** [comment]
### Sonar scan result
 - No new bug found. Congratulation!

More detail, see in https://sonarqube-dev.intgdc.com/dashboard?id=gooddata-ruby-gate-PR1717

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-sonarqube-scan-zuul-el7/13/ : SUCCESS in 2m 09s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-merge-compliance-check/493/ : SUCCESS in 24s
- https://checklist.intgdc.com/job/lcm/job/gooddata-ruby-travis-check/144/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-fossa-scanning-zuul-el7/54/ : FAILURE in 1m 19s (non-voting)

**Pull request merged**